routes/cache.js

```diff
@@ -71,4 +102,24 @@ router.put('/item', function (req, res) {
   })
 })
 
+// PUT multiple cache items
+router.put('/items', function (req, res) {
+  const items = JSON.parse(req.body)
+  const memcached = req.app.locals.memcached
+  if(!Array.isArray(items)) {
+    return res.status(400).json(jerr.BadRequest)
+  }
+
+  return Promise.allSettled(items.map(i => setPromise(memcached, i)))
```

Comment:
Should be simpler to use `utils.promisify` on `memcached.set` before getting into the `map` loop instead of calling `setPromise` for each item.

Reply:
Done! It is indeed simpler :-)